styles/base/_links.scss

```diff
@@ -39,7 +39,6 @@
   &:hover,
   &:active,
   &:focus {
-    transition: $sprk-link-transition;
```

Comment:
this transition is unnecessary because it's already happening in the base class`sprk-b-Link`. This is relevant to all the other places I've deleted transition.
![image](https://user-images.githubusercontent.com/4342363/118047073-3994c380-b348-11eb-9b47-979e4ed5ec6a.png)

Reply:
@RVMendoza, instead of deleting this it would be better for us to update it to match the other styles in this. So following the convention it would be `$sprk-link-simple-transition` and that would be set to the value of the default spark link. Having this allows the user to customize the transition as they are currently allowed to customize everything else.